src/app/qgsattributesformproperties.cpp

```diff
@@ -656,15 +647,15 @@ void QgsAttributesFormProperties::apply()
 
   QTreeWidgetItem *fieldContainer = mAvailableWidgetsTree->invisibleRootItem()->child( 0 );
 
-  int idx;
-
   for ( int i = 0; i < fieldContainer->childCount(); i++ )
   {
     QTreeWidgetItem *fieldItem = fieldContainer->child( i );
-    idx = fieldContainer->indexOfChild( fieldItem );
+    FieldConfig cfg = fieldItem->data( 0, FieldConfigRole ).value<FieldConfig>();
+
+    int idx = getFieldIndexByName( fieldItem->data( 0, FieldNameRole ).toString() );
 
-    QString name = mLayer->fields().at( idx ).name();
-    FieldConfig cfg = configForChild( idx );
+    //continue in case field does not exist anymore
+    if ( idx < 0 ) continue;
```

Comment:
Can you add a break before the continue? Just for the part of the code style that astyle still doesn't do ;)